pandas/tests/extension/base/reshaping.py

```diff
@@ -25,6 +25,21 @@ def test_concat(self, data, in_frame):
         assert dtype == data.dtype
         assert isinstance(result._data.blocks[0], ExtensionBlock)
 
+    @pytest.mark.parametrize('in_frame', [True])
```

Comment:
Should this be `[True, False]`?  If `False` isn't supported yet, might be nice to write a quick comment, as separate code paths for `False` exist in the test.

Reply:
Yeah, was debugging that case. Thanks.